tools/anonymizer_istex.js

```diff
@@ -0,0 +1,41 @@
+/*
+*  Tool to anonymize logs file
+*  It reads line by line the file and remove unwanted values
+*
+*/
+
+
+var fs = require('fs');
+var InputLogName;
+var OutputLogName;
+
+if(process.argv[2] != null){
+	InputLogName = process.argv[2];
+}else{
+	console.log('You need to pass in 1rst parameter, a valid path to the log file');
+	process.exit();
+}
+
+if(process.argv[3] != null){
```

Comment:
Rule of thumb : prefer strict equality to avoid surprises. Here, the comparison works because `undefined` is (weakly) equal to `null`, and `process.argv[3]` is `undefined`.